README.md

```diff
@@ -1,8 +1,23 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# personal-portfolio-site
 
-> This is my port folio site.
+# My Portfolio
+
+> Description the project.
+> This project contains my personal information and all the projects I have built using HTML and CSS and Javascript.
+> In this part of the project I designed the Hamburguer Menu, I added the navbar to it.
+
+
+## Project Tasks List (Working-in-progress)
+
+- [x] Setting up the initial project structure
+- [x] mobile version skeleton
+- [x] Contact form
+- [x] Desktop version
+- [x] Application deploy
+- [x] Evaluate accessibility
+- [x] Mobile Menu
+
```

Comment:
- [ ] Kindly add these sections filling them as much as possible to help someone wants to clone and run your project on live server:
 
- Getting Started
To get a local copy up and running follow these simple example steps.

- Prerequisites
- Setup
- Install
- Usage